packages/web/src/components/withdraw-usdc-modal/components/TransferSuccessful.tsx

```diff
@@ -52,9 +51,12 @@ export const TransferSuccessful = ({
 }) => {
   const { data: balance } = useUSDCBalance()
   const signature = useSelector(getWithdrawTransaction)
-  const balanceNumber = formatUSDCWeiToFloorCentsNumber(
-    (balance ?? new BN(0)) as BNUSDC
-  )
+  const balanceNumber =
```

Comment:
nit and probably out of scope for this pr but maybe we don't need to calc the balance here and can do it in the handler if that's the only place it's used... also seems like it's getting multiplied by 100 here just to get divided by 100 later ha